testsuite/cluster-test/src/instance.rs

```diff
@@ -271,6 +272,14 @@ impl Instance {
             )
             .await
     }
+
+    pub fn debug_interface_client(&self) -> AsyncNodeDebugClient {
```

Comment:
There is similar piece of code in `DebugPortLogWorker::new`:
```
let client = AsyncNodeDebugClient::new(http_client.clone(), instance.ip(), debug_interface_port);
```
Do you mind replacing it with your method there, e.g. replace this line with 
```
let client = instance.debug_interface_client();
```
To avoid code duplication.